server/controllers/userController.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express';
```

Comment:
I'll remove this for now because the actual user creation will work differently later